src/integrationTest/java/uk/gov/hmcts/reform/sendletter/controllers/CheckPrintStatusTest.java

```diff
@@ -73,25 +76,26 @@ public void should_return_204_when_there_are_no_unprinted_letters() throws Excep
         checkPrintStatus().andExpect(status().isNoContent());
 
         verify(letterRepository).getStaleLetters();
-        // TODO
-        // verify(insights, never()).trackNotPrintedLetter(captor.capture());
+        verify(insights, never()).trackNotPrintedLetter(captor.capture());
 
         assertThat(captor.getAllValues()).isEmpty();
     }
 
     @Test
     public void should_return_204_when_there_is_an_unprinted_letter() throws Exception {
-        setSentToPrintAt(createLetter(), secondBeforeFivePm);
+        UUID letterId = createLetter();
```

Comment:
I thought  `createLetter()` would return a Letter object.It might be not related but I would expect something as `UUID letterId = createLetter().getLetterId();`  or something similar or change method name?

Reply:
can change the name to un-confuse. as it is more of a integration test set up, loosely defined methods to keep it short and simple to read